src/imitation/policies/serialize.py

```diff
@@ -230,3 +230,14 @@ def _on_step(self) -> bool:
         output_dir = self.policy_dir / f"{self.num_timesteps:012d}"
         save_stable_model(output_dir, self.model)
         return True
+
+
+def _load_rollouts_from_huggingface(
```

Comment:
Why is this prefixed with `_` (private method) if it's being used from other modules?